### PR TITLE
Removed TODOs and FIXMEs. Adding new TODO/FIXME will fail CI.

### PR DESCRIPTION
**Pull request by asznajder**
Removed all TODOs and FIXMEs from code. 

I was unsure about two things connected with failing builds that are have TODO/FIXME.

1) Is so simple approach fine? Or maybe should our search be case insensitive? If so, the filter will have to be a little bit smarter since for example `toDoc` phrase appears in code.

2) Where TODO/FIXME are allowed? I bet that `.drone.yml` but what else? Is there any `.git` or `target` directory? In my local directory I could find TODO there.

### CODE REVIEW

**Comment by fommil** [comment]
Cool! Can you use git grep instead? Just exclude the drone defn itself.

File: .drone.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ build:
     - EMACS=/opt/emacs-24.5/bin/emacs
     - AKKA_TEST_TIMEFACTOR=10
   commands:
-    - if grep -rlE 'TODO|FIXME' . | grep -Ev 'target|.git|.drone.yml' > /dev/null; then
+    - if git grep -E 'TODO|FIXME' . | grep -v '.drone.yml' > /dev/null; then
```
Comment (fommil) [inline .drone.yml]: no need for the `.` in `git grep`. It knows where to search. Also, there is a more standard way to ignore files, see http://stackoverflow.com/a/14226610/1041691

Comment (fommil) [inline .drone.yml]: no need to throw anything to `/dev/null` it might be useful to see the matches.

Comment (asznajder) [inline .drone.yml]: Fixed, thanks!

**Comment by asznajder** [comment]
CI builds failed due something not connected with this commit, right?

**Comment by fommil** [comment]
thanks!

**Comment by fommil** [comment]
can you please raise a trivial PR with a TODO in a random file? Just to convince ourselves that it definitely works :smile:

**Comment by asznajder** [comment]
Sure!

**Pull request merged**